### PR TITLE
[WIP] Unittest for crowd-sourcing

### PR DESCRIPTION
**Pull request by SaketJNU**

### CODE REVIEW

File: f8a_jobs/handlers/test/test_crowd_source_tag.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    pkg_data = correct_data.get("result", {}).get("data", [])
+    assert (len(pkg_data) == 1)
+    for user_tag_data in pkg_data:
+        user_tags = user_tag_data.get("user_tags", [])
+        pkg_name = user_tag_data.get("name")[0]
+        assert (len(user_tags) == 2)
+        assert (pkg_name == "org.ongit:eclipse.jgit")
+        pt = []
+        for ut in user_tags:
+            utt = acs._process_tags(ut)
```
Comment (jpopelka) [inline f8a_jobs/handlers/test/test_crowd_source_tag.py]: Well, the `AggregateCrowdSourceTags._process_tags()` does not actually do any processing.
If you move the tags processing from `_read_tags_from_graph()` into that method, then we can test it here.

Comment (SaketJNU) [reply f8a_jobs/handlers/test/test_crowd_source_tag.py]: Thanks @jpopelka, I was also thinking on this point. I will write a separate method from ```_read_tags_from_graph()``` and will update PR's.

**Comment by jpopelka** [comment]
#219 has been merged, so you should be able to add unit tests (into [tests/](https://github.com/fabric8-analytics/fabric8-analytics-jobs/tree/master/tests)) and they should be run with each change in a pull request.

I'd suggest to add them directly to #211 instead of having them separately here.

**Comment by jpopelka** [comment]
You can also run the tests locally with `./runtest.sh`

**Comment by SaketJNU** [comment]
Hi @jpopelka, in order to add this unit-test through #211, we need to close this PR (216) and append the earlier one?

**Comment by jpopelka** [comment]
> in order to add this unit-test through #211, we need to close this PR (216) and append the earlier one?

Yes

**Comment by SaketJNU** [comment]
Thanks @jpopelka, I am closing the PR as these codes are appended with https://github.com/fabric8-analytics/fabric8-analytics-jobs/pull/211.

**Pull request closed without merging**